cl/users/views.py

```diff
@@ -655,3 +687,54 @@ def moosend_webhook(request: HttpRequest) -> HttpResponse:
     # Moosend does a GET when you create/edit the automation workflow,
     # so we need to return a 200 even for GETs.
     return HttpResponse("<h1>200: OK</h1>")
+
+
+@login_required
+@never_cache
+def view_webhooks(request: AuthenticatedHttpRequest) -> HttpResponse:
+    if request.method == "POST":
+        instance = Webhook()
+        form = WebhookForm(request.POST, instance=instance)
+        if form.is_valid():
+            instance.user = request.user
+            form.save()
+            return HttpResponseRedirect(reverse("view_api"))
```

Comment:
Does this `reverse` still make sense?

Reply:
yeah, changed it to `view_webhooks`, in the future this won't be needed since in the next PR I'll change this view to support htmx for webhooks.